conf_files/pocs.yaml

```diff
@@ -91,14 +91,14 @@ observations:
 # By default all images are stored on googlecloud servers and we also
 # use a few google services to store metadata, communicate with servers, etc.
 #
-# See $PANDIR/panoptes/utils/google/README.md for details about authentication.
+# See $POCS/pocs/utils/google/README.md for details about authentication.
```

Comment:
This shouldn't be changed, the readme comes from the `panoptes-utils` repository.